notebooks/amused-lightweight-text-to-image/amused-lightweight-text-to-image.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.            gr.Slider(0, 1000, label="Seed", step=1)
</pre><p>why do you reduce seed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
On my side, the slider widget behaves weird when its range is up to maxint value. For example, moving a slider by a pixel increases the seed value by many thousands. For me that was kind of weird, so I decided to lower the maximum number since there is only a limited number of possible values a slider can have anyway.

Reply:
got it, @aleksandr-mokrov could you please check reported by Nikita issue for amused gradio interface

Reply:
Yes, the behavior described is true. When we have a large range, the slider has a large increment of 10 million or so. We have such behavior in several notebooks. We discussed it in one of our PR and suggested that we should provide all available range to choose. It is possible to set any specific seed in the input box. If we reduce the range, then we will limit the choice. Perhaps it makes sense to set `step=1`, then moving the slider by one pixel will still change the value by a large number, but the step will not be a multiple of tens of millions and will not look so weird.

Reply:
<p>Reverted max value and kept step=1</p>